Create outer-misalignment.md

ArranMcC opened the pull request.

ArranMcC committed:
Create outer-misalignment.md

omarkohl commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,11 @@
+Intermediate
```
This is "Basic" not intermediate, which is good! IMHO "Basic" is the most important section. I really like what you wrote because it strikes the tone that I am aiming for myself :slightly_smiling_face: .

ArranMcC replied:
```suggestion
Basic
```

omarkohl commented on content/en/posts/outer-misalignment.md:
The hugo-specific file header is missing. Check the other posts. If you create the posts with "hugo new" they will be created automatically. It's also fine to insert them manually.

omarkohl commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,11 @@
+Intermediate
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you unintentionally teach it to fetch any paper-related item, including your important documents. This is a simple analogy for outer misalignment – the dog's action doesn't align with your true intentions.
```
I have to think about this some more and get clarity myself, but it sounds more like inner misalignment to me. The dog being confronted with other types of important documents sounds like an out of distribution event.

I'll get back to this. I might be confused.

omarkohl replied:
This point is still a little unclear to me. Do you feel you have clarity on this? i.e. do you feel confident that this is an example of outer misalignment? If yes, we can merge it and if necessary amend it later. Can you give me an example of the difference of outer and inner misalignment e.g. with the dog?

ArranMcC replied:
You are correct, good catch! The issue is the word 'unintentionally', which actually implies that this is an example of inner misalignment. If the dog was explicitly told to fetch all paper-related items, that would be outer misalignment. If the dog was told to fetch newspapers but misinterpreted that as all paper-related items, that would be inner misalignment. I'll try to edit it, but as I said on Signal earlier, I'm having problems with Github, especially when it comes to editing posts

ArranMcC replied:
I believe I've now edited the post

marverati replied:
Upon further reflection, I think the dog example is actually still a bit difficult - the problem here is that when training a dog, there isn't really this middle step where inner and outer misalignment would be differentiated. So the dog example could in a way be an example of both, and thus doesn't really help making the distinction between the two concepts clear.

marverati replied:
I changed the phrasing a bit now to make clearer that it's really about mistakes made by you as the trainer, rather than misgeneralized goals due to a distribution shift.

omarkohl commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,11 @@
+
+Relation to Other Terms
+
+While outer misalignment deals with the mismatch between human intent and the AI's specified objective, inner alignment deals with the discrepancy between the AI's specified objective and the AI's learned behavior. In our dog analogy, if the dog fetches the newspaper but then decides to chew it up (despite not being told to do so), it exhibits inner misalignment.
```
Same doubt as above. This sounds like something else, but I need to think about it.

omarkohl commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,11 @@
+
+Relation to Other Terms
+
+While outer misalignment deals with the mismatch between human intent and the AI's specified objective, inner alignment deals with the discrepancy between the AI's specified objective and the AI's learned behavior. In our dog analogy, if the dog fetches the newspaper but then decides to chew it up (despite not being told to do so), it exhibits inner misalignment.
```
I suggest leaving only the first sentence and then linking to "inner-misalignment". You can create a link as follows:

```
[Inner Misalignment]({{< ref "/posts/inner-misalignment" >}})
```

ArranMcC replied:
```suggestion
While outer misalignment deals with the mismatch between human intent and the AI's specified objective, [Inner Misalignment]({{< ref "/posts/inner-misalignment" >}}) deals with the discrepancy between the AI's specified objective and the AI's learned behavior.
```

omarkohl commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,11 @@
+Relation to Other Terms
+
+While outer misalignment deals with the mismatch between human intent and the AI's specified objective, inner alignment deals with the discrepancy between the AI's specified objective and the AI's learned behavior. In our dog analogy, if the dog fetches the newspaper but then decides to chew it up (despite not being told to do so), it exhibits inner misalignment.
+Reward misspecification is a subset of outer misalignment and pertains specifically to situations where the reward function (or the feedback mechanism) for an AI isn't properly defined. If the rewards provided to the AI don't encourage the right behaviors, the AI may optimize for unintended outcomes.
```
Same as above. I suggest leaving the first sentence and then linking.

ArranMcC replied:
```suggestion
[Reward Misspecification]({{< ref "/posts/reward-misspecification" >}}) is a subset of outer misalignment and pertains specifically to situations where the reward function (or the feedback mechanism) for an AI isn't properly defined. If the rewards provided to the AI don't encourage the right behaviors, the AI may optimize for unintended outcomes.
```

omarkohl reviewed (commented):
Thanks, great job!

ArranMcC committed:
Update content/en/posts/outer-misalignment.md

Changed intermediate to basic

ArranMcC committed:
Update content/en/posts/outer-misalignment.md

ArranMcC committed:
Update content/en/posts/outer-misalignment.md

ArranMcC committed:
Update outer-misalignment.md

ArranMcC commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,17 @@
+When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model is only as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
+For instance, consider a hypothetical AI tasked with minimizing the number of emails in an inbox. If not specified correctly, the AI might simply delete all emails (including the important ones) to achieve a 'zero' inbox count. The AI technically achieved its objective, but it was not aligned with the human user's true desires.
+
+Relation to Other Terms
```
```suggestion
Related Concepts
```

ArranMcC committed:
Update content/en/posts/outer-misalignment.md

ArranMcC commented:
@omarkohl I think I've now added the header and links, changed the level to basic and updated the section headign to related concepts. Is this post now ready to be approved?

ArranMcC commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,17 @@
+Basic
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you unintentionally teach it to fetch any paper-related item, including your important documents. This is a simple analogy for outer misalignment – the dog's action doesn't align with your true intentions.
```
```suggestion
Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you teach it to fetch any paper-related item, which could include your important documents. This is a simple analogy for outer misalignment – the instructions which you gave to the dog don't align with your true intentions.
```

ArranMcC committed:
Update content/en/posts/outer-misalignment.md

marverati commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,17 @@
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you teach it to fetch any paper-related item, which could include your important documents. This is a simple analogy for outer misalignment – the instructions which you gave to the dog don't align with your true intentions.
+When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model is only as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
```
```suggestion
When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model can only be as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
```

marverati replied:
I would make this small change, because our goal specification in a way defines an upper bound of the AI's performance. Inner misalignment may still mean its actual performance will be much worse than what the defined goal would in principle allow.

marverati commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,17 @@
+When we specify an objective for a machine learning model, we hope it understands and follows our intent. However, the model is only as good as the explicit instructions we provide. If there's a discrepancy between our intent and the specified goal, the AI might act in ways that are unexpected or harmful.
+For instance, consider a hypothetical AI tasked with minimizing the number of emails in an inbox. If not specified correctly, the AI might simply delete all emails (including the important ones) to achieve a 'zero' inbox count. The AI technically achieved its objective, but it was not aligned with the human user's true desires.
+
+Related Concepts
```
```suggestion
## Related Concepts
```

marverati commented on content/en/posts/outer-misalignment.md:
```diff
@@ -0,0 +1,17 @@
+Related Concepts
+
+While outer misalignment deals with the mismatch between human intent and the AI's specified objective, [Inner Misalignment]({{< ref "/posts/inner-misalignment" >}}) deals with the discrepancy between the AI's specified objective and the AI's learned behavior.
+[Reward Misspecification]({{< ref "/posts/reward-misspecification" >}}) is a subset of outer misalignment and pertains specifically to situations where the reward function (or the feedback mechanism) for an AI isn't properly defined. If the rewards provided to the AI don't encourage the right behaviors, the AI may optimize for unintended outcomes.
```
If reward misspecification is only a subset of outer misalignment, what would be an example of outer misalignment that is not reward misspecification? To me the two terms are almost synonymous, so would be happy to know if there's more to it. :)

marverati replied:
Given that this article: https://www.lesswrong.com/tag/outer-alignment claims that reward misspecification is basically equivalent to outer misalignment, I adjusted the phrasing now accordingly.

marverati commented:
Well written @ArranMcC, and this time no disagreements from my side, seems to hit the concept very well. :ok_hand:

marverati committed:
Adjustments to outer alignment article, and addition of intermediate level

marverati commented:
I made a few adjustments now and also added a proposal for the intermediate explanation.

marverati committed:
Add dog image to outer misalignment post

marverati committed:
Add inner misalignment post and cross-link between them
